crv/crvCurveMesh.cc

```diff
@@ -192,12 +193,12 @@ bool BezierCurver::run()
   }
 
   convertInterpolatingToBezier();
-
+/*
```

Comment:
@avinmoharana 

These are not commented on in the original branch.

Reply:
When you are on core's root directory and `bezier_fields` branch you can use

`git checkout develop -- crv/crvCurveMesh.cc` to revert this file back to what it is on the develop branch.